biodatasets/scicite/scicite.py

```diff
@@ -0,0 +1,212 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A dataset loader for the SciCite dataset.
+
+SciCite is a dataset of 11K manually annotated citation intents based on
+citation context in the computer science and biomedical domains.
+
+Some of the code in this module is based on the corresponding module in the
+datasets library.
+https://github.com/huggingface/datasets/blob/master/datasets/scicite/scicite.py
+
+In the source schema, we follow the datasets implementation and replace
+missing values.
+TODO: Use standard BigBio missing values.
+"""
+
+import json
+import os
+from typing import Dict, List, Tuple
+
+import datasets
+import numpy as np
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{cohan:naacl19,
+  author    = {Arman Cohan and Waleed Ammar and Madeleine van Zuylen and Field Cady},
+  title     = {Structural Scaffolds for Citation Intent Classification in Scientific Publications},
+  booktitle = {Conference of the North American Chapter of the Association for Computational Linguistics},
+  year      = {2019},
+  url       = {https://aclanthology.org/N19-1361/},
+  doi       = {10.18653/v1/N19-1361},
+}
+"""
+
+_DATASETNAME = "scicite"
+
+_DESCRIPTION = """\
+SciCite is a dataset of 11K manually annotated citation intents based on
+citation context in the computer science and biomedical domains.
+"""
+
+_HOMEPAGE = "https://allenai.org/data/scicite"
+
+_LICENSE = ""
```

Comment:
@stephenbach I checked the scicite repo, and it says the data is under Apache License 2.0. Would it be possible to have it here?

Reply:
```suggestion
_LICENSE = "Apache License 2.0"
```
Hi @ruisi-su IANAL, but I read that to mean that the code in the repo is Apache 2.0, but there's no license on the data itself, which is just linked from the README. Left the change as a suggestion if you want to commit it. Thanks!

Reply:
Thanks @stephenbach and @ruisi-su ! Let's leave the `_LICENSE` as empty for this merge due to the ambiguity.